x/lightclient/keeper/grpc_query_header_enabled_chains.go

```diff
@@ -0,0 +1,38 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/lightclient/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// HeaderSupportedChains implements the Query/HeaderEnabledChains gRPC method
+// It returns a list for chains that support block header verification.
+// Some chains in this list might be disabled which is indicated by the value of the `enabled` field.
+func (k Keeper) HeaderSupportedChains(c context.Context, req *types.QueryHeaderSupportedChainsRequest) (*types.QueryHeaderSupportedChainsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+	ctx := sdk.UnwrapSDKContext(c)
+
+	val, _ := k.GetBlockHeaderVerification(ctx)
```

Comment:
Should return error if not found

Reply:
1e5d62c979b52915ce73e5ef819d5a479ab8cd77